SQF/dayz_code/actions/plotManagement/plotAddFriend.sqf

```diff
@@ -15,7 +15,7 @@ _friends = _thePlot getVariable ["plotfriends",[]];
 _toAdd = [_friendUID,toArray _friendName];
 _inList = false;
 
-{ if ((_x  select 0) == (_toAdd select 0)) exitWith { _inList = true; }; } count _friends;
+{ if ((_x  select 0) == (_toAdd select 0)) exitWith { _inList = true; }; false } count _friends;
```

Comment:
@ebaydayz wanted me to change it because sometimes it can fail apparently, he will be able to.explain it better

On 22 August 2016 10:13:07 AM NZST, icomrade <notifications@github.com> wrote:
>> @@ -15,7 +15,7 @@ _friends = _thePlot getVariable ["plotfriends",[]];
>>  _toAdd = [_friendUID,toArray _friendName];
>>  _inList = false;
>>  
>> -{ if ((_x  select 0) == (_toAdd select 0)) exitWith { _inList =
>true; }; } count _friends;
>> +{ if ((_x  select 0) == (_toAdd select 0)) exitWith { _inList =
>true; }; false } count _friends;
>
>What exactly does this change do?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/EpochModTeam/DayZ-Epoch/pull/1747/files/986e359a4939375e1e4228a43ce5b67baa53c9c8..5b01cd23a5d030375d84179015b0b95cf2c032ec#r75604480

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.